swagger.json

```diff
@@ -2574,9 +2574,240 @@
                     }
                 }
             }
+        },
+        "/addresses": {
+            "get": {
+                "tags": [
+                    "Default", "Fleet"
+                ],
+                "summary": "/addresses",
+                "description": "Fetch all addresses/geofences for the organization. An address contains either a circle or polygon geofence describing the address boundaries.",
+                "operationId": "GetOrganizationAddresses",
+                "parameters": [
+                    { "$ref": "#/parameters/accessTokenParam" }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "List of addresses/geofences. For each address entry, the geofence field will only be populated with either 'circle' or 'polygon', depending on the geofence type.",
```

Comment:
The phrase `List of addresses/geofences` confuses me. Are the two separate entities? Could we simplify this to `List of addresses for the organization`?

Reply:
Sorry I think this reply got lost after the rebase:

I agree that addresses/geofences is a bit confusing, but we use this terminology to refer to addresses in the dashboard so customers will likely be expecting this same terminology in the API. I think we should change this to refer to just addresses but will need to change it everywhere, so I propose we have the API docs match the dashboard for now.

cc @samsarahq/design